core/merge.js

```diff
@@ -792,6 +792,24 @@ const needsFileidMigration = (
   fileid /*: ?string */
 ) /*: boolean %checks */ => existing.fileid == null && fileid != null
 
+const sideNames = new Set(['local', 'remote'])
+
+const detectSingleSide = doc => {
```

Comment:
Maybe move to `core/metadata`?

Reply:
Or even better rebase on #1674 and move to `./core/side`? :stuck_out_tongue:

Reply:
It takes a document as single argument. It's probably best in `core/metadata`.

Reply:
However, the `sideNames` Set is probably best in `core/side` :grin:

Reply:
Yeah, my bad, I was thinking about `sideNames` :+1: